lib/valkyrie/persistence/solr/queries/default_paginator.rb

```diff
@@ -1,16 +1,26 @@
 # frozen_string_literal: true
 module Valkyrie::Persistence::Solr::Queries
   # Acts as a null object representing the default case for paginating over solr
-  # results. Often only used for the first iteration of a loop.
+  # results. Often only used for the first iteration of a loop, or to retrieve
+  # all Documents in an index.
   class DefaultPaginator
+    # Default parameter for the next page in search results in a Solr query request
+    # @see https://lucene.apache.org/solr/guide/7_4/pagination-of-results.html
+    # @return [Integer]
     def next_page
       1
     end
 
+    # Default parameter for the number of documents in a page of search results in a Solr query request
+    # @see https://lucene.apache.org/solr/guide/7_4/pagination-of-results.html
```

Comment:
Same as above, `https://lucene.apache.org/solr/guide/pagination-of-results.html`?